doc/dependencies.md

```diff
@@ -65,56 +90,67 @@ flowchart TD
-    - Calls `transferFrom()`, and `approve()` on **IERC20 Token** for token operations
-    - Prepares cross-chain calls to **App Chain Gateway**'s `receiveParameters()` function via retryable tickets
+   - Calls `get(bytes[])` on **Settlement Chain Parameter Registry** to retrieve parameter values for bridging
+   - Calls `createRetryableTicket()` and `calculateRetryableSubmissionFee()` on **ERC20 Inbox** for cross-chain messaging
+   - Calls **FeeToken** and **USDC** for token operations
+   - Prepares cross-chain deposits to **App Chain Gateway**'s `receiveDeposit()` function via retryable tickets
+   - Prepares cross-chain calls to **App Chain Gateway**'s `receiveParameters()` function via retryable tickets
 
-2. **Node Registry**:
+2. **Payer Registry**:
 
-    - Calls `get(bytes)` on **Settlement Chain Parameter Registry** to retrieve admin, node manager, and migrator addresses
+   - Calls **FeeToken** and **USDC** for token operations
 
-3. **Payer Registry**:
+3. **PayerReportManager**:
 
-    - Calls `get(bytes)` on **Settlement Chain Parameter Registry** to retrieve settler, fee distributor, minimum deposit, withdraw lock period, and a migrator address
-    - Calls `balanceOf()`, `transfer()`, and `transferFrom()` on **IERC20 Token** for token operations
+   - Calls `getIsCanonicalNode()`, `getIsCanonicalNode()`, and `getSigner()` on **NodeRegistry** to retrieve canonical node information
+   - Calls `settleUsage()` on **PayerRegistry** to settle usage for batches of payer fees
 
-4. **Rate Registry**:
+4. **DistributionManager**:
 
-    - Calls `get(bytes)` on **Settlement Chain Parameter Registry** to retrieve message fee, storage fee, congestion fee, target rate per minute, and a migrator address
+   - Calls **FeeToken** and **USDC** for token operations
+   - Calls `ownerOf()` on **NodeRegistry** to verify node ownership
+   - Calls `sendExcessToFeeDistributor()` on **PayerRegistry** to trigger the **PayerRegistry** to send it its excess fee tokens
+   - Calls `getPayerReports()` on **PayerReportManager** to retrieve payer report state information
 
-### App Chain Contracts
+5. **DepositSplitter**:
 
-1. **App Chain Gateway**:
+   - Calls **FeeToken** and **USDC** for token operations
+   - Calls `deposit()` on **PayerRegistry** to deposit fee tokens into the **PayerRegistry**
+   - Calls `deposit()` on **SettlementChainGateway** to deposit fee tokens into the **SettlementChainGateway**
 
-    - Calls `set(bytes, bytes32)` on **App Chain Parameter Registry** to store parameters received from settlement chain
-    - Calls `get(bytes)` on **App Chain Parameter Registry** to retrieve a migrator address
+6. **FeeToken**:
 
-2. **Group Message Broadcaster**:
+   - Calls **USDC** for token wrapping operations
 
-    - Calls `get(bytes)` on **App Chain Parameter Registry** to retrieve min/max payload sizes, pause status, and a migrator address
+### App Chain Contracts
 
-3. **Identity Update Broadcaster**:
```

Comment:
where did these 2 go?

Reply:
Gone because it is covered by the general statement above `"All app chain contracts interact with the app chain parameter registry to retrieve their configuration parameters."` and that is it's only interaction, which is the same for the `RateRegistry`.